### ISSUE TITLE
Serenity parallel execution documentation link (gradle specific)

### ISSUE THREAD

**Issue by prashantrbs** [serenity-bdd/serenity-core#2806]
Hi Everyone,

i am using serenity bdd and Gradle as a build automation tool. i have 250plus test cases which are taking a lot of time to execute so i want to work on parallel execution. i have searched Gradle specific parallel execution can you please provide the documentation link.

Thanks in advance

**Comment by wakaleo** [comment]
https://docs.gradle.org/current/userguide/java_testing.html#sec:test_execution

**Comment by prashantrbs** [comment]
Thank you.

**Comment by iholub** [comment]
are there more details how to use it? 
I set maxParallelForks = 2 in my gradle build file
but still my cucumber scenarios are executed in sequence (not parallel)

**Comment by prashantigs** [comment]
Yes even i tried this its not working at all. i have cucumber, JUnit and gradle in my framework. someone please assist with this with detail document. above document does not help. Thanks.
